src/workbooks/governance/workbook.json

```
@@ -753,6 +746,136 @@
                         },
                         "name": "Compute networking"
                       },
+                      {
+                        "type": 1,
+                        "content": {
+                          "json": "# Managed Disks utilization"
```

This looks like the same change as the other PR. Does one of these include everything? If so, close the PR that's only partial. If not, you'll have some merge conflicts to contend with.
```suggestion
                          "json": "# Managed disks utilization"
```